### ISSUE TITLE
No Issue, Just Giving Thanks

### ISSUE THREAD

**Issue by raycadle** [JoeMrCoffee/NextCloudDockerCompose#2]
Didn't have any issues with the compose file, at least none that you didn't cover. I just wanted to say, thank you! Your video on YT was the only one that made sense, and Nextcloud was up and running within minutes. Keep up the good work man!

**Comment by JoeMrCoffee** [comment]
Really glad this helped! Thanks for the pull request around the innodb compression issue. I believe that issue is gone when using MariaDB 10.5, but let me know if you still see the issue and I'll verify on my end as well. Thanks!

**Comment by raycadle** [comment]
You're absolutely right about it being a non-issue on version 10.5 with the latest nextcloud image. I guess I've got to make a little more changes to my config. Thanks for letting me know!

**Comment by JoeMrCoffee** [comment]
Thanks for the suggestion - I'm new to Github as well, and I didn't think to add the compression off flag in the Docker Compose file. When I first came across the issue I think I was on an earlier version of MariaDB, and then I later upgraded to 10.5 late last year. I decided to leave the workaround in the Readme in case anyone had an issue. Glad it's working on your version. Thanks again for the comment and glad you found it helpful.
